src/Microsoft.ML.Transforms/Text/NgramTransform.cs

```diff
@@ -668,9 +667,27 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
     }
 
     /// <summary>
-    /// Produces a bag of counts of ngrams(sequences of consecutive values of length 1-n) in a given vector of keys.
-    /// It does so by building a dictionary of ngrams and using the id in the dictionary as the index in the bag.
+    /// Produces a vector of counts of ngrams (sequences of consecutive words) encountered in the input text.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes |
+    /// | Input column data type | Vector of [Keys](<xref:Microsoft.ML.Data.KeyDataViewType>) |
+    /// | Output column data type | Known-sized vector of <xref:System.Single> |
+    ///
+    /// The resulting [NgramExtractingTransformer]<xref:Microsoft.ML.Transforms.Text.NgramExtractingTransformer>
```

Comment:
>NgramExtractingTransformer] [](start = 23, length = 27)

If you make it as link you should put <xref> in braces `[text](url)`

Reply:
And i'm not sure we even want make it url, what is wrong with just `<xref>`

---
In reply to: [276868333](https://github.com/dotnet/machinelearning/pull/3418#discussion_r276868333) [](ancestors = 276868333)

Reply:
If the anchor text is the same as the class/type name, you don't really need to use [anchor](xref:UID) pattern. You can save some time by using autolinks: <xref:UID>


---
In reply to: [276868428](https://github.com/dotnet/machinelearning/pull/3418#discussion_r276868428) [](ancestors = 276868428,276868333)